app/oauth2-server-config/client/admin/views/oauthApp.html

```diff
@@ -22,7 +22,7 @@
 						<div class="input-line double-col">
 							<label>{{_ "Redirect_URI"}}</label>
 							<div>
-								<input type="text" class="rc-input__element" name="redirectUri" value="{{data.redirectUri}}" />
+								<textarea class="rc-input__element" name="redirectUri" rows="4" style="height: auto">{{data.redirectUri}}</textarea>
```

Comment:
Multiple URIs are being split with a comma, despite how I have written, which leads to a weird behavior since the value is updated right after I hit "save".

can you please force multiple URIs to be separate by a new line?